app/src/main/java/org/fossasia/openevent/app/event/attendees/AttendeesPresenter.java

```diff
@@ -37,11 +37,6 @@ public void attach() {
     }
 
     @Override
-    public void detach() {
```

Comment:
Don't remove detach. We MUST detach presenter at some point, it is giving NPE just means it is detach at wrong lifecycle method